widget.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Login-andrey",
```

Comment:
@AndrRyaz for the future, please do not commit these changes because they end up being in our commit log.